test/parallel_api/experimental/asynch.pass.cpp

```diff
@@ -14,19 +14,19 @@
 //===----------------------------------------------------------------------===//
 
 #include "oneapi/dpl/execution"
-#include "oneapi/dpl/async"
 #include "oneapi/dpl/iterator"
 
 #include "support/pstl_test_config.h"
 
+#if TEST_SYCL_PRESENT
+#   include "oneapi/dpl/async"
```

Comment:
I think this header should be under the `TEST_DPCPP_BACKEND_PRESENT` macro

Reply:
Can confirm it solved the issues.